packages/material-ui-styled-engine-sc/package.json

```diff
@@ -44,6 +44,9 @@
   "peerDependenciesMeta": {
     "@types/styled-components": {
       "optional": true
+    },
+    "styled-components": {
+      "optional": true
```

Comment:
```suggestion
```

Reply:
For clarity, It seems that styled-components shouldn't be optional because `@material-ui/styled-engine-sc` is not a dependency of `@material-ui/core`. By making it nonoptional we can display a warning in the console if forgotten.

Did I miss something?